Marlin/src/feature/fwretract.cpp

```diff
@@ -202,6 +226,9 @@ void FWRetract::retract(const bool retracting
   // If swap retract/recover update the retracted_swap flag too
   #if EXTRUDERS > 1
     if (swapping) retracted_swap[active_extruder] = retracting;
+    else retracted[active_extruder] = retracting;
+  #else
+    retracted[active_extruder] = retracting;              // Active extruder now retracted / recovered
   #endif
```

Comment:
@Weruminger — Notice that `retracted[active_extruder] = retracting` is already done on line 224. Therefore the `else` clause and extra instance here is pointless.

Reply:
@thinkyhead  "Thank you for the very quick review, title changing and code maintainment."

I see! Line 224 is too much , it adds the default retraction|resume to the swap retraction|resume. the result does not harm anything, but is not what there is expected. In case of swap retraction|resume, only feasible with multiple extruders, there shall be a decision to just retract|resume the swap feed. 

thx. I will change this.